contents/docs/self-host/deploy/snippets/upgrading.mdx

```diff
@@ -29,5 +29,5 @@ If you see this error
 Error: UPGRADE FAILED: release posthog failed, and has been rolled back due to atomic being set: post-upgrade hooks failed: warning: Hook post-upgrade posthog/templates/migrate.job.yaml failed: jobs.batch "posthog-migrate" already exists
 ```
 
-it happens because the migrate job was left around from a previous upgrade attempt, we just need to kill that job (`kubectl delete job posthog-migrate -n posthog`) before running the upgrade again.
+it happens because the migrate job was left around from a previous upgrade attempt, we just need to kill that job (`kubectl -n posthog delete job $(kubectl -n posthog get jobs --no-headers -o custom-columns=NAME:.metadata.name)`) before running the upgrade again.
```

Comment:
I worry if we in the future add some jobs that potentially shouldn't be killed (e.g. think async migrations becomes a job that runs without blocking upgrades somehow) & we forget to update this here.

Reply:
In that case I think what you wrote will be true also without this change (here we only changed the job name, adding a timestamp at the end)

Reply:
not sure I follow, before we explicitly said only `posthog-migrate`, now we run delete over all jobs in the namespace, no?

Reply:
Yes